### DISCUSSION TITLE
Import library in SharedWorker

### DISCUSSION THREAD

**Discussion by SmashPlusUltra** [webpack/webpack#15791] [Q&A]
I need to add [signalR]( https://www.npmjs.com/package/@microsoft/signalr) using absolute path, so doing it in the SharedWorker I've received a lot of errors/warnings.
I'm probably missing something. 

Setup:
webpack.config.js
```
const path = require('path');
module.exports = {
    mode:'development',
    entry: './src/index.js',
    output: {
        filename: 'bundle.js',
        path: path.join(__dirname, 'public')
    },    
    module: {
        rules: [{
          loader: 'babel-loader',
          test: /\.js$/,
          exclude: /node_modules/
        }]
      },
      resolve:{
        alias:{
            SignalR: path.resolve(__dirname, 'src/scripts/')
        },            
      },      
}

```
Output:
The output is quite long so If needed I'll copy fully
![image](https://user-images.githubusercontent.com/53838441/167317301-c38d2a7e-45f0-4232-929b-d587262eb1c7.png)

**Comment by vankop** [comment]
this warnings means that webpack can't resolve module request. e.g. first warning means that in `./node_modules/ws/lib/buffer-util.js` there is request for `bufferutil`, but webpack can't find it

**Reply by SmashPlusUltra** [reply]
But, this type of library should be imported as "externals"? so we don't get these errors?

**Reply by vankop** [reply]
I don't see this in your webpack config..
